elide-core/src/main/java/com/yahoo/elide/security/ChangeSpec.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2015, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.security;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+/**
+ * ChangeSpec for a particular field.
+ *
+ * @param <T> type parameter
+ */
+@AllArgsConstructor
+public class ChangeSpec<T> {
+    @Getter private final String parentType;
```

Comment:
I am still thinking about this. This is both a weird name and type for this; when I say "parent" I mean parent of the field and not parent resource. Perhaps we should pass the resource directly? It's highly likely you could have collisions here.

Consider writing an ID check that does something special based on type. For instance, we allow ID 0 for object `X` and a 1 for object `Y`. The check could not then distinguish.

Perhaps the correct solution is to forget and require a separate check be written.